tutorials/calibration_flow/gptq_like.py

```diff
@@ -0,0 +1,263 @@
+"""
+Traditional calibration flow has the following flow (see static_quant.py for code examples):
```

Comment:
@msaroufim added a bit more context here, please let me know if this makes sense